src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
```

Comment:
Reporter: PMD
Rule: EmptyIfStmt
Severity: WARN
File: /home/bjerre/workspace/violations-test/src/main/java/se/bjurr/violations/lib/example/OtherClass.java L10
 

Empty Code http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/empty.html#EmptyIfStmt

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>